en/docs/reference/connectors/cdc-inbound-endpoint/cdc-inbound-endpoint-example.md

```diff
@@ -0,0 +1,154 @@
+# CDC Inbound Endpoint Example 
+
+The CDC (Change Data Capture) inbound endpoint enables you to capture changes made to data in a database. Consider a scenario where the database crashes at some point, leading to the loss of data related to certain operations. This highlights the need to synchronize the data with a cloud-based backup storage. To minimize risk, updating the cloud storage in real time is preferable to batch updates. Therefore, it is important to detect modification events and update the cloud storage accordingly. The CDC inbound endpoint can be used for this purpose.
+
+## What you'll build
+
+This scenario demonstrates how the CDC inbound endpoint can be used to capture changes occurring in a MySQL database. In this example, we will configure the **CDC Inbound Endpoint** to capture changes made to a **MySQL** table, such as insertions, updates, and deletions.
```

Comment:
Why `MySQL` is in blod text in some places while others are not?

Reply:
It was unintentional. I'll update the text.